packages/text-area/test/text-area.test.js

```diff
@@ -347,4 +371,91 @@ describe('text-area', () => {
       expect(spy.callCount).to.equal(0);
     });
   });
+
+  describe('web platform pattern validation spec', () => {
```

Comment:
No need to mention WPT here, there is a link below.

```suggestion
  describe('pattern', () => {
```